### PR TITLE
Create pure3.md

### PR DESCRIPTION
**Pull request by jhdalek55**
This is very much a work in progress and at this point is not even complete. I crafted this from what had been posted in Standard PR #231 and Standard Issue #227. I frankly was not sure what we should do in terms of how we release the information or a patch once a vulnerability is confirmed. 

@hexsecs and others, please add on.

### CODE REVIEW

**Comment by mnm678** [comment]
Github has a good process for releasing CVEs and patches that allows them to be worked on in secret by a select group of maintainers and contributors, but then made public when they're ready (including comments and other context around the patches). That should work for updating the standard.

The challenge is that if there is a critical vulnerability that would affect implementations, we need a way to know who maintains those implementations in order to communicate the vulnerability before it is made public. With TUF we are able to communicate to all of the open source implementations, but for Uptane we don't have access to source code and so would need some kind of invite-only list of implementers (which sounds very challenging to maintain).

**Comment by jhdalek55** [comment]
Agreed.

File: pure3.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## *Reporting a vulnerability*
+Suspected security vulnerabilities should be reported to `uptane-security [AT] googlegroups [DOT] com. We're committed to working with security researchers to resolve the vulnerabilities they discover. You can help us by following these guidelines:
+
+Please give as much detail as possible for a suspected vulnerability in Uptane including:
+
+* Version in which it was found
+* Description of vulnerability
+* Examples (if applicable)
+
+Give us a reasonable time to correct the issue before you disclose the vulnerability. We care deeply about security, but we're an open-source project and our team consists mostly comprised of volunteers.
```
Comment (pattivacek) [inline pure3.md]:
```suggestion
Give us a reasonable time to correct the issue before you disclose the vulnerability. We care deeply about security, but we're an open-source project and our team consists mostly of volunteers.
```

**Comment by pattivacek** [comment]
I agree, I don't know easy it would be for us to really maintain such a list. The best we could do is have a voluntary signup for early disclosures. Also consider that if we become aware of vulnerabilities in the Standard, we have no way of knowing which implementations might include that vulnerability. Implementers may have intentionally or unintentionally avoided the problem. But I think we still need to make an effort here even if it is a challenge. Best to have the reporting framework in case we need it than to have nothing at all if an emergency comes up.

**Comment by jhdalek55** [comment]
I'm doing some additional work on this PR. Comments and edits are still wanted but understand that I am adding text.

**Comment by jhdalek55** [comment]
I've added a proposed procedure. Please review and add on.

**Comment by jhdalek55** [comment]
After our discussion at the 9/13 Standards meeting, it was decided to close this PR. We feel that Uptane should not be in the business of collecting and resolving vulnerabilities. Instead, we need to encourage individuals to report vulnerabilities found in applications using Uptane to whoever released the application. Instead, we are adding information on how to submit errata  for the Standard. See Standard PR #227

**Pull request closed without merging**